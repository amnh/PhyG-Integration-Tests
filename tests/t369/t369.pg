--input t25.tre, swap(drift:3, acceptequal:2.0), inputting Hennig86 files with all character settings included
set(seed:1634561640)
read(rename: "woman_rename.txt")
--woman_allchars.ss includes cc+, cc-, continuous, ?, [ ], /N 
read(tnt: "woman_allchars.ss")
read(tnt: "woman_step.ss")
read(include:"Include_woman.txt")
read(newick: "t25.tre")
swap(drift:3, acceptequal:2.0)
report("t369.csv", data, overwrite)
report("t369.tre", graphs, newick, overwrite)
