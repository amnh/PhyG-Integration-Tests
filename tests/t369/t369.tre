(AlbanianTosk:0.0,((((Yiddish:41.1,(Faroese:26.0,Scots:11.0)HTU20:31.1)HTU21:38.6,(ArmenianWestern:13.0,Cornish:27.0)HTU27:63.0)HTU19:23.1,(Manx:25.5,((Assamese:34.6,SanskritVedic:14.1)HTU23:5.0,((EasternFrisianSaterland:9.9,(Icelandic:75.0,KolschColognian:7.3)HTU30:22.0)HTU25:15.9,(AncientIonicGreek:25.3,Danish:36.9)HTU28:11.0)HTU26:11.0)HTU24:25.5)HTU22:15.0)HTU18:36.0,(Irish:3.6,OldPrussian:2.4000000000000004)HTU29:34.4)HTU17:0.0)HTU16[1228.9];
