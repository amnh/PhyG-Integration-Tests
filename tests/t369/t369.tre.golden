(Afrikaans:145.7,(Welsh:145.15,(OldPrussian:73.15,(Samogitian:168.75,(Breton:169.3,((ArmenianEastern:28.35,TocharianB:79.69999999999999)HTU73:23.85,((Latgalian:216.4,(ClassicalArmenian:168.25,((Manx:179.75,(AlbanianTosk:53.65,TocharianA:38.550000000000004)HTU54:41.15)HTU53:11.799999999999999,(OldIrish:173.05,(Irish:215.25,ScottishGaelic:172.55)HTU71:7.15)HTU70:25.15)HTU69:15.950000000000001)HTU68:7.65)HTU56:50.15,(Cornish:181.95000000000002,(ArmenianWestern:105.05,(Lithuanian:201.9,(Latvian:168.85,((((SouthernTsakonian:47.85,((Danish:39.35,(PennsylvaniaDutch:53.25,(EnglishAmerican:30.15,(CentralFranconianRipuarian:168.65,Gothic:168.54999999999998)HTU91:16.65)HTU64:47.35)HTU59:28.15)HTU79:31.15,((GreekModern:51.65,((AncientGreek:167.85,(PharasaGreek:108.45,Yiddish:113.65)HTU80:9.65)HTU60:11.85,((Dutch:156.75,(OldHighGerman:20.15,(Luxemburgish:61.65,SchwyzerdutschSwissGerman:62.85)HTU96:33.25)HTU58:16.15)HTU63:29.55,(Norweigan:105.95,Swedish:95.25)HTU94:20.15)HTU82:11.35)HTU83:31.75)HTU62:29.65,(LowSaxon:31.85,(KolschColognian:33.95,(Icelandic:66.15,(German:31.35,NorthFrisianOomrang:126.05)HTU97:36.65)HTU95:30.65)HTU90:27.65)HTU67:14.5)HTU89:29.85)HTU92:9.15)HTU65:17.35,(AncientIonicGreek:25.35,(EnglishBritish:44.0,(Faroese:46.65,Scots:88.65)HTU87:34.5)HTU84:40.65)HTU85:38.25)HTU66:20.65,(SanskritVedic:44.25,(Assamese:30.65,(Avestan:23.1,Luwian:122.45)HTU98:9.75)HTU76:22.2)HTU88:24.65)HTU61:10.1,(EasternFrisianSaterland:106.55,WesternFrisian:23.85)HTU81:21.05)HTU86:38.15)HTU72:17.950000000000003)HTU75:34.15)HTU77:30.65)HTU74:49.15)HTU93:19.0)HTU78:9.65)HTU55:17.85)HTU57:39.75)HTU51:61.05)HTU52:8.7)HTU50[2510.5000000000005];
